go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/openshift/api v0.0.0-20200311183032-85e16cc5dd7c
 	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
 	github.com/openshift/crd-schema-gen v1.0.0
-	github.com/openshift/installer v0.9.0-master.0.20190726121806-6e8f9c335410 // indirect
+	github.com/openshift/installer v0.9.0-master.0.20190726121806-6e8f9c335410
```

Comment:
@coreydaley why do we need an installer bump when the version listed in `go.mod` hasn't changed? I also don't see a corresponding change in `go.sum`.

Reply:
IT was changed from being indirect to direct

Reply:
the ciro would not compile without that update and pulling the installer back in as a direct dependency

Reply:
because we use installer types in pkg/storage/util/util.go